manifests/management.pp

```diff
@@ -1,14 +1,16 @@
 # Management Server.
+class rancher::management(
+  $rancher_manager_port = '8080',
+){
 
-class rancher::management {
+  require ::docker
```

Comment:
I meant to ask about the explicit top scope i.e. why `require ::docker` and not `require docker`?  I see people doing this and I haven't been able to find out why.  I'll go through and do it consistently if we agree this is preferred.